02_tidyverse.qmd

```diff
@@ -450,13 +513,12 @@ count(x = asec, pernum, offpov)
 
 \newpage
 
-# Mutating joins
+## Mutating joins
```

Comment:
I added joins content in the advanced data cleaning section (https://awunderground.github.io/data-science-for-public-policy2/02_advanced-data-cleaning.html). We should reconcile these sections.

Reply:
resolved by (1) adding filtering joins to the intro chapter and (2) moving that content to the review of the advanced chapter